models/unit.go

```diff
@@ -141,10 +141,10 @@ func loadUnitConfig() {
 func (u UnitType) UnitGlobalDisabled() bool {
 	for _, ud := range DisabledRepoUnits {
 		if u == ud {
-			return true
+			return false
```

Comment:
this make things worse ... I'll have a look at the creation menue and fix it ... sory I know you have put in time and effort

Reply:
This disables the global projects from the ui. How does it make it worse?

Reply:
@6543 Sorry, I will fix the UI template. I'll try to do it today or at least tomorrow.